### PR TITLE
DSP-13836 bump versions of cassandra driver and netty

### PR DESCRIPTION
**Pull request by jtgrabowski**

### CODE REVIEW

File: project/Versions.scala [hunk 1]
```diff
@@ -15,12 +15,12 @@ object Versions {
   lazy val logback = "1.0.7"
   lazy val mesos = sys.env.getOrElse("MESOS_VERSION", "1.0.0-2.0.89.ubuntu1404")
   lazy val metrics = "2.2.0"
-  lazy val netty = "4.0.42.Final"
+  lazy val netty = "4.0.44.Final"
```
Comment (jacek-lewandowski) [inline project/Versions.scala:18]: 4.1 ?

**Review by jacek-lewandowski — COMMENTED** [review]
Just one comment - we probably need to bump Netty to 4.1 since we do it for Spark. This may follow bumping Akka to some newer release which works with Netty 4.1.

File: project/ExclusionRules.scala [hunk 2]
```diff
@@ -6,6 +6,7 @@ object ExclusionRules {
   val excludeScalaTest = ExclusionRule(organization = "org.scalatest")
   val excludeScala = ExclusionRule(organization = "org.scala-lang")
   val excludeNettyIo = ExclusionRule(organization = "io.netty", artifact = "netty-all")
+  val excludeNettyOrg = ExclusionRule(organization = "io.netty")
```
Comment (jacek-lewandowski) [inline project/ExclusionRules.scala]: i find this name confusing, but this is not really important

**Review by jacek-lewandowski — APPROVED** [review]

**Comment by jacek-lewandowski** [comment]
+1

File: project/ExclusionRules.scala (hunk 2 above)
Comment (jtgrabowski) [reply project/ExclusionRules.scala]: true, will change

File: project/Versions.scala (hunk 1 above)
Comment (jtgrabowski) [reply project/Versions.scala:18]: DSP-14152

**Pull request merged**